docs/usage/model_config.md

```diff
@@ -8,6 +8,9 @@ Options:
 **`anystr_strip_whitespace`**
 : whether to strip leading and trailing whitespace for str & byte types (default: `False`)
 
+**`anystr_lower`**
+: whether to make all characters lowercase for str & byte types (default: `False`)
```

Comment:
we should also add a comment about this to [the docs](https://pydantic-docs.helpmanual.io/usage/types/#constrained-types).

Best of all would be a description of all the arguments to `constr` like we do [for URLs](https://pydantic-docs.helpmanual.io/usage/types/#urls)

Reply:
Done. Let me know what you think

Reply:
I'm not sure if this is the best place to ask this but why do we need `curtail_length` when we have `max_length`? 

They seem to do the same thing except I don't fully understand it.

Reply:
`curtail_length` does not raise an Error if the string is too long but just shrinks it to the set value

Reply:
I see. thanks.